web/src/main/java/com/futurewei/alcor/web/entity/vpc/VpcEntity.java

```diff
@@ -19,6 +20,9 @@
     @JsonProperty("routes")
     private List<RouteEntity> routeEntities;
 
+    @JsonProperty("router")
```

Comment:
@kevin-zhonghao @cj-chung 

For VPCEntity, could use extract the common fields of VPCEntity and NetworkEntity (not existed yet) to a common class? VPCEntity and NetworkEntity could inherit this common class. This way, we could clearly see the differences in two different scenarios. 

Not high priority for this PR. We could discuss and plan in next PR.

Reply:
That's a good idea ~ we could discuss later